index.php

```diff
@@ -6,6 +6,12 @@
 
 $recipeDao = new RecipeDao();
 
+if (isset($_POST['recipe-name'])){
+    $new_recipe = new Recipe($_POST['recipe-name'], $_POST['recipe-description'], $_POST['recipe-date']
```

Comment:
Only thing that's needed is some extra validation here - to check that the data the user submit is actually valid.

It's not a huge job, but it is a little bit time-consuming.